### ISSUE TITLE
docs!

### ISSUE THREAD

**Issue by jeka-kiselyov** [FizzyFlow/Flow#10]
Most important, as there're many classes extending EventEmmiter - is to have some doc files with information about events emitting by each class there. But general docblocks for methods whould be cool to.

**Comment by jeka-kiselyov** [comment]
As part of this research - write out some convention for events naming/params, as there's some mess already with it.

**Comment by banguit** [comment]
Yes, I think we need to cover all API by JSDoc. Because currently it's looks a little bit messy.
